tests/utils/coordinates.test.js

```diff
@@ -8,7 +8,7 @@ suite('utils.coordinates', function () {
     });
 
     test('rejects invalid coordinate', function () {
-      assert.ok(coordinates.isCoordinate('1 1 2.5 -3'));
+      assert.ok(!coordinates.isCoordinate('1 1 2.5 -3'));
```

Comment:
This test was correct. A string with four values "1 1 2.5 -3" is a valid coordinate